City-of-Helsinki/kukkuu-ui
# KK-661 | Remove browser validation from preliminary form

## Description

Removes the browser validation from the preliminary form as well, which I forgot to do initially.

Fixes a few other unrelated problems:
* Errors appearing incorrectly in forms. This was caused by component from a newer version of HDS not working as expected with Grid layouts
* Overflowing time picker in event detail view
* Occurrence info wrapping incorrectly in occurrence detail view

## Context

[KK-661](https://helsinkisolutionoffice.atlassian.net/browse/KK-661)

## How Has This Been Tested?

I've manually checked that the form is not validated with browser validation.

## Manual Testing Instructions for Reviewers

As a non logged in user

1. Input birth date 04-02-2006 and Helsinki on the front page form and accept terms
1. Submit the form
1. Expect to see a frowny face

## Screenshots
